<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VideoSyncTestWindow</class>
 <widget class="QMainWindow" name="VideoSyncTestWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VideoSyncTest</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1">
    <item>
     <widget class="PhVideoView" name="_videoView" native="true"/>
    </item>
    <item>
     <widget class="PhMediaPanel" name="mediaController" native="true"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuOpen_recent">
     <property name="title">
      <string>Open recent</string>
     </property>
     <addaction name="actionEmpty"/>
    </widget>
    <addaction name="actionOpen"/>
    <addaction name="menuOpen_recent"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionEmpty">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Empty</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PhMediaPanel</class>
   <extends>QWidget</extends>
   <header>PhCommonUI/PhMediaPanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PhVideoView</class>
   <extends>QWidget</extends>
   <header>PhVideo/PhVideoView.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
